.gitmodules

```diff
@@ -2,7 +2,12 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
+# [submodule "deps/power-grid-model"]
+# 	path = deps/power-grid-model
+# 	url = git@github.com:PowerGridModel/power-grid-model.git
+# 	branch = main
+
```

Comment:
please delete deprecated stuff
```suggestion
```

Reply:
I think I have a question here. My last PR other reviewers suggested using ssh, so I had it as an ssh format. But with SSH i am having some problems when trying to update the submodule. Thus, I changed it to the https format and it updated easily. Do you care whether it is ssh or https format?

Reply:
moving to HTTPS is not a solution to your problems. It's because you have not set up your Git environment correctly. Please follow the steps in https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent?platform=windows

That said, in general, since we are open source, we do not (and should not) demand from our users that they have SSH set up. In fact, someone may just download our code and everything should work just like that. Hence, using HTTPS is better **IN THIS REPO**. Note: if you are working for inner-source/company-internal projects, you should always use SSH instead of HTTPS because it adds another layer of security.